packages/subproviders/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "5.1.0-beta.2",
```

Comment:
Wasn't quite sure what the correct version number should be here, can someone confirm? CC @fabioberger

Reply:
This is correct! Thanks for checking!